app/views/layouts/_ae_resolve_options.html.haml

```diff
@@ -1,123 +1,18 @@
-- field_changed_url ||= "form_field_changed"
+- field_changed_url ||= "resolve_automate_simulation"
```

Comment:
Can this field_changed_url variable be removed now? Since the react forms no longer need this we usually remove it but I see its used in the url variable below. Can you look into if this can be removed as well as the actual form_field_changed function in miq_ae_tools_controller.rb?